enterprise/backend/src/metabase_enterprise/content_translation/api/dictionary.clj

```diff
@@ -4,12 +4,49 @@
    [clojure.data.csv :as csv]
    [clojure.java.io :as io]
    [clojure.string :as str]
-   [metabase.util.i18n :refer [tru]]
+   [metabase.util.i18n :as i18n :refer [tru]]
    [metabase.util.log :as log]
    [toucan2.core :as t2]))
 
 (set! *warn-on-reflection* true)
 
+(def ^:private http-status-unprocessable 422)
+
+(def ^:private max-string-length 255)
+
+(defn- row-has-correct-number-of-fields
+  "Checks if a row has the expected format with exactly 3 columns."
+  [row]
+  (and (vector? row) (= (count row) 3)))
```

Comment:
`count` will work with any sequential so I don't think you need to do the `vector?` test unless there's somewhere you're doing index based access.